### PR TITLE
SCRNS-3256 Setup Looping Content using franklin

### PR DESCRIPTION
**Pull request by div8063**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@aem-screens/screens-offlineresources-generator",
-    "version": "0.0.1",
+    "version": "0.0.1-HG-019",
```
Comment (absarasw) [inline package.json:3]: I think we should keep it as 0.0.1.

**Comment by absarasw** [comment]
@div8063, Can you please remove the lint errors if there is any? Also remove the util files in src directory if we are moving the utils in different folder.

File: src/channelHtmlGenerator/channelHtmlGenerator.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+            name: element.children[1].children[0].data,
+            link: element.children[3].children[0].data
+          })
+        }
+      } catch (err) {
+        console.warn(`Invalid word doc row`, err);
+      }
+    });
+    if (sheetDetails.length === 0) {
+      // fetch from sharepoint format now
```
Comment (habansal) [inline src/channelHtmlGenerator/channelHtmlGenerator.js:71]: @div8063 Why do we need different handling of html generated via google doc v/s sharepoint?
Why is that differentiation not needed in the Block code -
https://github.com/hlxsites/inside-adobe-screens/commit/a0da7d41e272d7ede339e6a831ba6efca32bcd7a#diff-0ba135df53c420d893ac5474ae71abe07027087aba7d760a4534f1d585cefd4bR166

File: src/createManifest.js [hunk 3]
```diff
@@ -99,23 +104,29 @@ export default class ManifestGenerator {
...
+      const {
+        scripts = '[]', styles = '[]', assets = '[]',
+        inlineImages = '[]', dependencies = '[]'
+      } = data;
+      const scriptsList = JSON.parse(scripts);
+      const stylesList = JSON.parse(styles);
+      const assetsList = JSON.parse(assets);
+      assetsList.forEach(ManifestGenerator.trimImagesPath);
+      const inlineImagesList = JSON.parse(inlineImages);
+      inlineImagesList.forEach(ManifestGenerator.trimImagesPath);
```
[the remark this answers is not in the document]
Comment (div8063) [reply src/createManifest.js:122]: Hi @absarasw 
We have provided access to media_ extensions on cdn config, so this should not be required anymore right?

[the remark this answers is not in the document]
Comment (div8063) [reply src/createManifest.js:126]: As discussed, this issue is a general one with manifest creation and not specific to inside-adobe. However, when we fix the manifest generation, we would need changes in the html generator as well.
Keeping this change for next iteration.

**Review by absarasw — DISMISSED** [review]

File: src/channelHtmlGenerator/channelHtmlGenerator.js (hunk 2 above)
Comment (div8063) [reply src/channelHtmlGenerator/channelHtmlGenerator.js]: Updated the sheet details extraction logic at both places to use only one format.
The user manual will be updated with details on the format for carousel block syntax.

**Review by absarasw — APPROVED** [review]

**Pull request merged**